frontend/src/routes/(authenticated)/user/+page.ts

```diff
@@ -16,9 +16,13 @@ export const load = (({ url }) => {
   const emailResult = url.searchParams.get('emailResult') as EmailResult | null;
   if (emailResult) {
     if (!EMAIL_RESULTS.includes(emailResult)) throw new Error(`Invalid emailResult: ${emailResult}.`);
-    if (browser) void goto(`${url.pathname}`, { replaceState: true });
+    if (browser) {
+      void goto(`${url.pathname}`, { replaceState: true });
```

Comment:
are we sure this isn't what's causing issues? this is just to remove the search param from the url to keep it pretty right? If that's the case maybe we should just remove it.

Reply:
Very good idea 😆.
I just removed the troublesome, unnecessarily complicated, minor UX feature.